Desktop

Robison-faike123 opened the pull request:
- [ ]  This is the  dekstop version looks like the one in the Figma design template one that I chose. 
- [ ] The Figma template I chose is one of those which has complications mentioned like the one with full name and first name and last name ,so please don't consider it important  (for first name and last name)  Please ignore this part of the design template and  I used only one input for the full name for all versions.
- [ ] please am requesting if you are not going to approve it ,give me direction on what you want me to do because we are time bad am late on deadline and I have done this alone without help so please be considerate

Robison-faike123 committed:
added files and images

Robison-faike123 committed:
updated changes as required

Robison-faike123 committed:
update on footer

Robison-faike123 committed:
update files

Robison-faike123 committed:
added files in css

Favourezeugwa commented on index.html:
```diff
@@ -11,18 +11,27 @@
       <div class="AppBar">
-        <header class="ToolBar">
+        <div class="ToolBar">
           <nav>
             <img src="images/NormalButton.png" alt="" class="NormalButton" />
           </nav>
           <div class="Logo">
             <a href="#">Faishar Faike</a>
           </div>
-        </header>
+          <ul class="nav-menu">
+            <li><a href="#portfolio">Portfolio</a></li>
+            <li><a href="#contact">Contact</a></li>
+            <li><a href="#About">About</a></li>
+       </ul>
+        </div>
```
- I think it will be great to make the content in our nav bar a bit more centered just like on figma.

Here is what I can see compared to figma 👇🏽 
### Your navbar

![navbad](https://user-images.githubusercontent.com/99246359/200112203-432464cc-564d-4749-81a8-d526b46481a2.PNG)

### Figma

![corrfig](https://user-images.githubusercontent.com/99246359/200112221-cf12f734-1af4-42a5-895a-34f667c6e7d3.PNG)

- I think you may have styled the ul in a `.nav-menu` style in your CSS, Kindly fix it and the logo as well 🌸

Robison-faike123 replied:
thanks for observation am going to consider it

Favourezeugwa commented on styles.css:
```diff
@@ -672,3 +714,638 @@ button:hover {
+    height: 769.27px;
+    top: -140px;
+  }
+
+  .content {
+    display: flex;
+    flex-direction: row-reverse;
+    padding-top: 250px;
+  }
+
```
- I think it will be better if we reduce the excess padding added to your `.content` style. Currently i believe the space at the top and bottom o this section is a bit too much. 🤗 

Here is what I mean 👇🏽 

## Your design
![faikeexce](https://user-images.githubusercontent.com/99246359/200112493-a9ef3794-b4e6-4ccc-825e-155afdbb7e16.PNG)

## Figma

![dieqfig](https://user-images.githubusercontent.com/99246359/200112568-3334f10a-3d84-483c-b6af-eccca68753d4.PNG)

How about we start first by reducing the `padding-top: 250px;`  to `padding-top: 100px;` then search for anywhere else you may have added some excess bottom padding or margin 🥂

Robison-faike123 replied:
thanks for elaborated review and recommendation is on spot consider it done as you suggested

Favourezeugwa replied:
Sure @Robison-faike123 , you are welcome 🤗

Favourezeugwa commented on styles.css:
```diff
@@ -672,3 +714,638 @@ button:hover {
+    width: 369px;
+    border: no border;
+  }
+
+  #w4 {
+    background-image: url("/images/Img desktop@2x.png") !important;
+    background-repeat: no-repeat;
+    height: 435px;
+    width: 369px;
+    margin-top: 20px;
+  }
+
```
- I think the `margin-top: 20px;` added to your fourth card has made it look a bit bigger/longer than the other cards

Here 👇🏽 
![cardss](https://user-images.githubusercontent.com/99246359/200112842-85f5a92b-2da9-457e-9a23-7a8b9e14c77d.PNG)

- I believe if we removed the `margin-top: 20px;`, it will fit properly and all cards will be equally aligned 🍒

Robison-faike123 replied:
thanks for that it was try and error method which resulted into forgetting that

Favourezeugwa reviewed (changes requested):
## STATUS: REQUIRES CHANGES 🚦 ⚠️ ♻️ 

Hi @Robison-faike123 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![woman so close](https://user-images.githubusercontent.com/99246359/200113046-4b371861-4c6e-4a9a-b260-fed789cb234f.gif)


## Highlights 💯 
- No Linter errors ✔️ 
- Professional Readme ✔️ 
- Good commit messages ✔️ 
- GitHub flow used ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Robison-faike123 commented:
Is it all ? Thanks for your feedback it worth and straight to the point let
me work on it now

On Sat, 5 Nov 2022, 1:26 p.m. Favour Ezeugwa, ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
> STATUS: REQUIRES CHANGES 🚦 ⚠️ ♻️
>
> Hi @Robison-faike123 <https://github.com/Robison-faike123> ,
>
> Good job so far!
> There are some issues that you still need to work on to go to the next
> project but you are almost there!
>
> [image: woman so close]
> <https://user-images.githubusercontent.com/99246359/200113046-4b371861-4c6e-4a9a-b260-fed789cb234f.gif>
> Highlights 💯
>
>    - No Linter errors ✔️
>    - Professional Readme ✔️
>    - Good commit messages ✔️
>    - GitHub flow used ✔️
>
> Required Changes ♻️
>
> *Check the comments under the review.*
> Optional suggestions
>
> *Every comment with the [OPTIONAL] prefix is not crucial enough to stop
> the approval of this PR. However, I strongly recommend you to take them
> into account as they can make your code better.*
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
> ------------------------------
>
> In index.html
> <https://github.com/Robison-faike123/4ne-first/pull/5#discussion_r1014608489>
> :
>
> > +        <div class="ToolBar">
>
>            <nav>
>
>              <img src="images/NormalButton.png" alt="" class="NormalButton" />
>
>            </nav>
>
>            <div class="Logo">
>
>              <a href="#">Faishar Faike</a>
>
>            </div>
>
> -        </header>
>
> +          <ul class="nav-menu">
>
> +            <li><a href="#portfolio">Portfolio</a></li>
>
> +            <li><a href="#contact">Contact</a></li>
>
> +            <li><a href="#About">About</a></li>
>
> +       </ul>
>
> +        </div>
>
>
>
>    - I think it will be great to make the content in our nav bar a bit
>    more centered just like on figma.
>
> Here is what I can see compared to figma 👇🏽
> Your navbar
>
> [image: navbad]
> <https://user-images.githubusercontent.com/99246359/200112203-432464cc-564d-4749-81a8-d526b46481a2.PNG>
> Figma
>
> [image: corrfig]
> <https://user-images.githubusercontent.com/99246359/200112221-cf12f734-1af4-42a5-895a-34f667c6e7d3.PNG>
>
>    - I think you h=may have styled the ul in a .nav-menu style in your
>    CSS, Kindly fix it and the logo as well 🌸
>
> ------------------------------
>
> In styles.css
> <https://github.com/Robison-faike123/4ne-first/pull/5#discussion_r1014609370>
> :
>
> > +
>
> +  .content {
>
> +    display: flex;
>
> +    flex-direction: row-reverse;
>
> +    padding-top: 250px;
>
> +  }
>
> +
>
>
>
>    - I think it will be better if we reduce the excess padding added to
>    your .content style. Currently i believe the space at the top and
>    bottom o this section is a bit too much. 🤗
>
> Here is what I mean 👇🏽
> Your design
>
> [image: faikeexce]
> <https://user-images.githubusercontent.com/99246359/200112493-a9ef3794-b4e6-4ccc-825e-155afdbb7e16.PNG>
> Figma
>
> [image: dieqfig]
> <https://user-images.githubusercontent.com/99246359/200112568-3334f10a-3d84-483c-b6af-eccca68753d4.PNG>
>
> How about we start first by reducing the padding-top: 250px; to padding-top:
> 100px; then search for anywhere else you may have added some excess
> bottom padding or margin 🥂
> ------------------------------
>
> In styles.css
> <https://github.com/Robison-faike123/4ne-first/pull/5#discussion_r1014610081>
> :
>
> > +
>
> +  #w4 {
>
> +    background-image: url("/images/Img ***@***.***") !important;
>
> +    background-repeat: no-repeat;
>
> +    height: 435px;
>
> +    width: 369px;
>
> +    margin-top: 20px;
>
> +  }
>
> +
>
>
>
>    - I think the margin-top: 20px; added to your fourth card has made it
>    look a bit bigger/longer than the other cards
>
> Here 👇🏽
> [image: cardss]
> <https://user-images.githubusercontent.com/99246359/200112842-85f5a92b-2da9-457e-9a23-7a8b9e14c77d.PNG>
>
>    - I believe if we removed the margin-top: 20px;, it will fit properly
>    and all cards will be equally aligned 🍒
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Robison-faike123/4ne-first/pull/5#pullrequestreview-1169383973>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZK3RBKTS4YPQYQNIHNRYX3WGYR5NANCNFSM6AAAAAARXZ2LIA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Robison-faike123 committed:
required changes fixed

Robison-faike123 commented:
- [ ] as required I have fixed all the suggested first review said it was okay and on point 
- [ ] I had to reduce the padding-top of content and height of headline which I did .
- [ ] I was told to remove the margin-top on card 4 which I have done

ichala reviewed (approved):
Hi @Robison-faike123 ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
![68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966](https://user-images.githubusercontent.com/85113567/184604589-430cf4bc-1d2e-4603-976a-7a48a899ea35.gif)

# To Highlight 💪🏻

-  Changes Implemented✔️ 



 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
